### ISSUE TITLE
Bug when trying to move to first row in table

### ISSUE THREAD

**Issue by fisher729** [mltony/nvda-tonys-enhancements#5]
Hi.

STR:
1. Visit https://www.spoilertv.com/2021/06/ratings-for-monday-31st-may-network.html?utm_source=Trending&utm_medium=TrendingToday
2. Wait for a second, then press T for table.
3. Press Alt+Control+PageUp

Expected: NVDA moves to the first row in the table.
Actual: NVDA freezes for a while, then says "Cannot find edge of table in this direction.

Also, the following error is in the log.
error executing script: <bound method GlobalPlugin.injectTableFunction.<locals>.<lambda> of <virtualBuffers.gecko_ia2.Gecko_ia2 object at 0x079EBA10>> with gesture 'alt+ctrl+page up'
Traceback (most recent call last):
  File "scriptHandler.pyc", line 208, in executeScript
  File "C:\Users\andre\AppData\Roaming\nvda\addons\tonysEnhancements\globalPlugins\tonysEnhancements.py", line 1199, in <lambda>
    script = lambda self,gesture: function(self, gesture, *args, **kwargs)
  File "C:\Users\andre\AppData\Roaming\nvda\addons\tonysEnhancements\globalPlugins\tonysEnhancements.py", line 607, in findTableCell
    info = self._getTableCellAt(tableID, self.selection,origRow, origCol)
NameError: name 'self' is not defined

Using: NVDA 2020.4, Windows 10, Firefox 89.0
Add-on: Tony's Enhancements 1.10
Other instance: when reading from the top of the column with NVDA+Control+Shift+Down Arrow.

**Comment by CyrilleB79** [comment]
The error is not present anymore with the native control+alt+upArrow gesture in NVDA 2022.2beta2. Thus, I think this issue can be closed when #13 is fixed, i.e. when the control+alt+pgUp/pgDown/Home/End are unmapped in the add-on, at least with NVDA 2022.2 onwards.
